docs/site/migration/tables/lb3-only-commands.html

```diff
@@ -0,0 +1,104 @@
+<head>
+  {% include analytics.html %}
+</head>
+
+<table width="750">
+  <thead>
+    <tr>
+      <th width="200">Commands</th>
+      <th>Description</th>
+      <th>Workaround in LB4</th>
+    </tr>
+  </thead>
+  <tbody>
+
+    <tr>
+      <td>
+        <code>lb bluemix</code>
+      </td>
+      <td> Create a new application destined in Bluemix
+      </td>
+      <td> We have several tutorials on how to deploy a LoopBack 4 application to the IBM Cloud: <a href="https://loopback.io/doc/en/lb4/Deploying-to-IBM-Cloud.html">Deploying to Cloud Foundry on IBM Cloud</a>, <a href="https://loopback.io/doc/en/lb4/deploying_to_ibm_cloud_kubernetes.html">Deploying to Kubernetes on IBM Cloud</a>, <a href="https://loopback.io/doc/en/lb4/Appsody-LoopBack.html">Deploying to Kubernetes on IBM Cloud using Appsody</a>
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        <code>lb property</code>
+      </td>
+      <td>
+        Adds properties to an existing model
+      </td>
+      <td> Open the model file, and manually add a new property.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        <code>lb acl</code>
+      </td>
+      <td>
+        Adds acl settings to models
+      </td>
+      <td> Use the <a href="https://loopback.io/doc/en/lb4/Loopback-component-authentication.html">AuthenticationComponent</a> and <a href="https://loopback.io/doc/en/lb4/Loopback-component-authorization.html">AuthorizationComponent</a> to set up authentication and authorization manually.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        <code>lb remote-method</code>
+      </td>
+      <td>
+        Adds a new remote method to a model
+      </td>
+      <td> Open the model's controller file and add a new method manually. Depending on the purpose of this new controller method, you may also need to a new method to the model's repository.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        <code>lb middleware</code>
+      </td>
+      <td>
+        Add new middleware to the application
+      </td>
+      <td> There is <a href="https://github.com/strongloop/loopback-next/pull/5118">work in progress</a> to enable express middleware in the form of interceptors that you will need to create and register
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        <code>lb boot-script</code>
+      </td>
+      <td>
+        Creates a skeleton boot script in server/boot directory. Allows for customized intialization of anything during boot phase
+      </td>
+      <td> Create a <a href="https://loopback.io/doc/en/lb4/Life-cycle.html#overview">lifecycle observer</a> to run custom code during different phases of the application's lifecycle; including startup.
```

Comment:
probably mention the migration guide for boot script in https://loopback.io/doc/en/lb4/migration-boot-scripts.html